supporting_content/metrics/amortizedcost.md

```diff
@@ -0,0 +1,71 @@
+
+
+###### Note 
+*Billed cost is showing the total cash outlay for the reservation. Also note there is no initial Reserved Instance fee representing the cost of the RI for the applicable period.*
+
+
+
+##### Also on each hour a line item is added as follows
+
+| Charge period           | Billing period           | Charge Type | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | ----------- | ----- | ----------- | -------------- |
+| Jan 1 2023 - Jan 2 2023 | Jan 1 2023 - Jan 31 2023 | Usage       | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*The amortization is equal to the (Upfront fee / (number of normalized units reserved x periods) x normalized line item units consumed).*
+
+___
+
+##### If the Reserved Instance is unused during a period, for example another ineligible instance type is used:
+
+
+| Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*For the On Demand usage the Billed cost and Amortized cost are the same with an Amortization of Zero. Since the RI went completely unused during the period there is an unused RI usage line item with the an amortization equal to (Upfront fee / (number of normalized units reserved x periods) x normalized line item units. Amortized Cost is equivalent to Amortized since this is a full upfront reservation*
+
+---
+
+##### If the RI is partially unused during a period, for example a scale down occurs:
+
+
+| Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*For the Reserved Instance Usage the Billed cost is zero as this RI is full upfront. It is especially important to note that the usage units must be normalized for this formula to be applicable in cases of size flexibility (e.g. c7g.large normalized units is 4, c7g.medium is 2). Amortized cost is the same as amortization because the upfront fee covers all of the compute usage expense for this line item.*
+
+
+
+#### Example usage scenarios
+
+See [Cost Metrics Examples Spreadsheet](https://docs.google.com/spreadsheets/d/1bhRELDgf3LTSfQJRrCyovTt65g4ElimYHq6fmKOz83E) for examples of billing data for various scenarios.
```

Comment:
Should this file be included in the Supporting Content as the google account might not always be available?

Reply:
Yeah I think we are going to have to handle providing examples outside of an google sheet. @udam-f2  what do you want to do her. I can make a table for examples as I did above but references to google docs may not be what we want to do.